src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -654,7 +654,7 @@ export class InputDatePicker
         ? endWrapper || startWrapper
         : startWrapper || endWrapper;
 
-    connectFloatingUI(this, this.referenceEl, this.floatingEl);
+    requestAnimationFrame(() => connectFloatingUI(this, this.referenceEl, this.floatingEl));
```

Comment:
Need to wait for next frame to connect floating UI because some rendering is still going on. The rendering needs to happen first before trying to get all scrollable parents.

Reply:
Interesting. On a related note, I was thinking `input-date-picker` should be refactored to use `popover` for consistency and reusability. Maybe that would help eliminate since this doesn't appear to be an issue with the `input-time-picker`.